<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContourDlg</class>
 <widget class="QDialog" name="ContourDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>184</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>184</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>184</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Contour Line Setting</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>150</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>24</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>150</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>24</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="defaultValueButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Restore Defaults</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="intervalLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Interval:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="ravineGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>42</y>
     <width>180</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Index Contour</string>
   </property>
   <widget class="QLabel" name="ravineColorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ravineThickLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Thick:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ravineThickSpin">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>55</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="ravineColorButton">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>25</y>
      <width>30</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="mainCurveGroup">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>42</y>
     <width>180</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Intermediate Contour</string>
   </property>
   <widget class="QLabel" name="mainColorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainThickLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Thick:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mainThickSpin">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>55</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="mainColorButton">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>25</y>
      <width>30</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="intervalEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="intervalUnitLabel">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>10</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>(20 ~ 10000m)</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>intervalEdit</tabstop>
  <tabstop>ravineColorButton</tabstop>
  <tabstop>ravineThickSpin</tabstop>
  <tabstop>mainColorButton</tabstop>
  <tabstop>mainThickSpin</tabstop>
  <tabstop>defaultValueButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
